src/samplers/tcp/bpf.c

```diff
@@ -85,4 +85,4 @@ int trace_tcp_rcv_state_process(struct pt_regs *ctx, struct sock *skp)
 
     start.delete(&skp);
     return 0;
-}
```

Comment:
Agree with adding a newline here, but I'd maybe keep it out of this PR.

Reply:
I think this was an auto format that snuck in when I was looking at some of the other files for examples. I've reverted it.